src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -97,14 +97,16 @@ class ProjectCreationModel {
 
     private fun createProject(selectedCollection: Collection) {
         creationUseCase
-                .newProject(selectedCollection, targetLanguage)
+                .newProject(selectedCollection, targetLanguage!!)
```

Comment:
Not sure how critical it is since targetLanguage should never be null at this point, but maybe add a line or two to verify non null and avoid `!!`? Might just be preference

Reply:
Yeah sounds good